index.html

```diff
@@ -19,7 +19,7 @@
                 <li><a href="#">About</a></li>
```

Comment:
Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1.
![image](https://user-images.githubusercontent.com/62290545/187484154-5e53abe4-63be-4bf7-a918-ced7518a332f.png)

- [x] You can use [this tool](https://www.aditus.io/button-contrast-checker/) to check on the color contrast ratio.